.github/workflows/semgrep-lint.yml

```diff
@@ -1,4 +1,4 @@
-name: sgrep-lint
+name: semgrep-lint
```

Comment:
Can this be changed to just semgrep or is there a naming collision

Reply:
oh, right! somehow my brain just find and replace'd the word, didn't notice this was the CLI command's name before